pulp_2to3_migration/app/plugin/deb/pulp_2to3_models.py

```diff
@@ -449,6 +449,10 @@ def create_pulp3_content(self):
             relative_path=self.package_relative_path,
             sha256=self.package_sha256,
         ).first()
+        if not package:
+            # Perhaps the package was never created because the file was corrupt!
+            # We simply don't create the PackageReleaseComponent and hope for the best.
+            return ???
```

Comment:
What can i return to make sure that the unit is eventually marked as migrated in the `Pulp2Content` table, but simply does not create any corresponding Pulp 3 unit?

Reply:
You can `return None, None` https://github.com/pulp/pulp-2to3-migration/blob/master/pulp_2to3_migration/app/plugin/content.py#L285

If it is not critical to create PackageReleaseComponent in case the corresponding package has not been migrated/save then this approach should be fine.

Reply:
PackageReleaseComponent simply associates a Package with a ReleaseComponent (for the sake of generating repo metadata). If the package is not migrated/created/saved, then this association does not make any sense and should not happen either. The resulting migrated repository simply won't contain the corrupt package.

Of course, I should go ahead and test this by deliberately corrupting a package before migrating.